### ISSUE TITLE
Don't add to playlist & playback new tracks from file browser

### ISSUE THREAD

**Issue by intranauta** [jfchapman/VUPlayer#5]
Hi, wonderful update (I'm since years a huge fan of the classic 2.49! It was my main player for a decade). 
(Sorry for my english, I'm spanish-speaker).

Now I installed this new version on windows 10 (x64), but I have a problem: when I double-click from the file browser on the music track (mp3, flac, etc.) or select "open with" [vuplayer], it open the player but don't add the track(s) to the playlist nor does it start playback (in 2.49, it automatically deleted the existent listed files and added the file (s) to a new playlist, starting playback automatically).

**Comment by jfchapman** [comment]
There's no support for opening files from Windows yet, but it's next on my list of things to do.
I will also be adding file browsing into the application itself.

**Comment by intranauta** [comment]
Oh, ok, I now understanding the "playlist" concept! Also the "add folder" function, sorting the tracks by its metadata. This allows you to navigate everything inside the vuplayer, it's fine. :) Yeah, file browsing into the app could be nice also, to contiue navigating your own folder structure.

Another thing that I found:; if I rename a folder or some files already added, the vuplayer of course no longer reads them. Is it possilble to think on the future in a function for "update" or "refresh" what has already been added (i.e. in the right-click menu on each Album)? It could also be updating over all the added tracks, but it would take a long time: I just added folders for 35GB and everything worked ok! But it takes a while, of course.

Thank you for the great work!
